modules/animation/src/animationcontroller.cpp

```diff
@@ -258,6 +258,50 @@ void AnimationController::stop() {
     eval(currentTime_, Seconds(0));
 }
 
+void AnimationController::nextKeyframe() {
```

Comment:
This functionality should be in animation, something like, getNextTime(time t). Maybe with a direction also.

Reply:
That would avoid the expensive getAllTimes.

Reply:
I.e. the functionallity the find the next time, not the evaluation

Reply:
Fixed, although I don't think there is any real performance gain here. The changes needed to be propagated to the tracks and sequences.